src/sentry/static/sentry/app/views/projectsDashboard/chart.tsx

```diff
@@ -1,22 +1,117 @@
 import React from 'react';
 
 import {Project} from 'app/types';
-import {Series} from 'app/types/echarts';
 import {t} from 'app/locale';
-import MiniBarChart from 'app/components/charts/miniBarChart';
+import BaseChart from 'app/components/charts/baseChart';
+import theme from 'app/utils/theme';
+
+type BaseChartProps = React.ComponentProps<typeof BaseChart>;
 
 type Props = {
-  stats: Required<Project>['stats'];
+  stats: Project['stats'];
+  transactionStats?: Project['transactionStats'];
 };
 
-const Chart = ({stats = []}: Props) => {
-  const series: Series[] = [
-    {
-      seriesName: t('Events'),
-      data: stats.map(point => ({name: point[0] * 1000, value: point[1]})),
+const Chart = ({stats, transactionStats}: Props) => {
+  const series: BaseChartProps['series'] = [];
+
+  if (transactionStats) {
+    series.push({
+      cursor: 'normal' as const,
+      name: t('Transactions'),
+      type: 'bar',
+      data: transactionStats.map(point => [point[0] * 1000, point[1]]),
```

Comment:
If we named the tuples, it might be nice to destructure the map callback arg so we can give them more specific names instead of referencing by index.